python/user_error.py

```diff
@@ -10,7 +11,7 @@ class UserError(Exception):
     :raises: ValueError (varsSoFar gets pickled into JSON, which may result in any number of errors depending on what types are inside)
     """
 
-    def __init__(self, exc_obj, exc_tb, varsSoFar={}, execTime=0):
+    def __init__(self, exc_obj: BaseException, exc_tb: TracebackType, varsSoFar={}, execTime=0):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing docstring in __init__](https://app.codacy.com/manual/almenon214/AREPL-backend/pullRequest?prid=5181101)